### ISSUE TITLE
Nova URL

### ISSUE THREAD

**Issue by alxferraz** [paoloo/servicos#1]
Olá @paoloo! mudei hoje a url pra nova e nada..... alguma idéia do que poderia ser? até ontem tudo estava rodando normalmente.

**Comment by paoloo** [comment]
Sim, desligaram o servidor, porque descobrimos o novo esquema. Aparentemente estão de olho no post do blog. Assim que lançarem uma atualização no **apk**  o serviço será decodificado.

**Comment by alxferraz** [comment]
Engraçado que o app deles ta funcionando ainda normalmente..... será que bloquearam algum dos dados fakes que enviamos junto com a requisição?

**Comment by alxferraz** [comment]
ideal seria essas conjecturas num outro thread privado neh?!

**Comment by paoloo** [comment]
É possivel. Tem como jogar uma proxy no celular, armazenar o request e me passar? Assim entendo o que tem de diferente e corrijo. Mas sei que mudaram o endereco, porque esta dando 404.

**Comment by paoloo** [comment]
bom, o pessoal está discutindo na thread do blog, então, pior que isso fica
dificil haha

Em 20 de novembro de 2015 20:09, André Luiz Ferraz <notifications@github.com
> escreveu:

> ideal seria essas conjecturas num outro thread privado neh?!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/paoloo/servicos/issues/1#issuecomment-158552837>.
>



-- 
Paolo Oliveira
CACTμS - Célula Avançada de Computação, Telefonia e Metodologias de
Segurança, Líder
LAMP - Laboratório de Aplicações Massiçamente Paralelas, Membro
LIT - Laboratório de Inovação Tecnológica, Membro
IFCE - Instituto Federal do Ceará

**Comment by williamokano** [comment]
Achei este código dentro do decompilado, porém não está abrindo, mas talvez seja um caminho

C2607o c2607o = new C2607o("http://soap.ws.placa.service.sinesp.serpro.gov.br/", "getStatus");

**Comment by alxferraz** [comment]
me add no novo repo! @alxferraz!

**Comment by williamokano** [comment]
Existe um novo repositório? Se tiver me add tbm @williamokano

**Comment by devmob2you** [comment]
APK has decompiled then find it:
/sinesp-cidadao/ConsultaPlacaNovo02102014

But, the responsse is:
<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">soap:Bodysoap:Fault<faultcode>soap:Client</faultcode><faultstring>Unmarshalling Error: unexpected element (uri:"", local:"placa"). Expected elements are &lt;{}a> </faultstring>/soap:Fault/soap:Body/soap:Envelope

Add me on the new repo to I work too.

Thanks.

**Comment by joaocalixto** [comment]
me add tb, please @joaocalixto

**Comment by macsousa** [comment]
por favor se puder me add ao novo repositorio @macsousa

**Comment by lucasromanojf** [comment]
Por favor me add também @lucasromanojf

**Comment by leonardocorrea1000** [comment]
Por favor me add também @leonardocorrea1000

**Comment by andersonbfn** [comment]
+1 please @andersonbfn

**Comment by jeorfevre** [comment]
me add tambem por gentileza @jeorfevre

**Comment by edgardksb** [comment]
+1 por favor @edgardksb

**Comment by lucazFerreira** [comment]
@paoloo por favor, me adicione no novo repositório? Estou com o seguinte erro, após mudar o Post para : http://sinespcidadao.sinesp.gov.br/sinesp-cidadao/ConsultaPlacaNovo02102014
Error: unexpected element (uri:"", local:"placa"). Expected elements are &lt;{}a> </faultstring>

**Comment by Mephisty31** [comment]
Se puder me add ao novo repositorio @Mephisty31 Valeu!

**Comment by MagnusIIIBR** [comment]
Me add ao novo repositório @MagnusIIIBR

**Comment by marcelocecin** [comment]
mudou novamente: http://sinespcidadao.sinesp.gov.br/sinesp-cidadao/mobile/consultar-placa

**Comment by marcelocecin** [comment]
chave parece que mudou para TRwf1iBwvCoSboSscGne
todavia a forma de calcular o token tbm

**Comment by marcelocecin** [comment]
o grupo do telegram está ativo ? ou é um novo repo ?

**Comment by rafaelrabaco** [comment]
Se existir o grupo ainda, quero entrar, se não eu posso criar...

a chave é essa mesmo @marcelocecin TRwf1iBwvCoSboSscGne, mas o calculo da hash mudou, estou tentando descobrir, mas sem sucesso ainda...


Unica coisa que consegui extrair da lib onde faz o calculo da hash, foi isso:

int Java_br_gov_sinesp_cidadao_android_FragmentConsulta_b(int arg0, int arg1, int arg2) {
    stack[2041] = r4;
    sp = sp - 0x24;
    r3 = *arg0;
    r4 = arg0;
    r3 = *(r3 + 0x2a4);
    r6 = (r3)(arg0, arg2, 0x0, r3, arg0, arg1, stack[2041], r5, r6, r7, r8, r10, lr);
    r8 = malloc(0x1e);
    r7 = malloc(0x1e);
    r5 = malloc(0x1e);
    s0 = malloc(0x1e);
    r2 = s0;
    getPackageName(r4, arg1, r2, r3);
    if (strcmp("br.gov.sinesp.cidadao.android", s0) == 0x0) {
            strcpy(r5, r6);
            strcat(r5, "TRwf1iBwvCoSboSscGne");
    }
    else {
            r5 = "abc123";
    }
    s0 = strlen(r5);
    r0 = strlen(r6);
    r3 = r0 - 0x0;
    if (r3 > 0x0) {
            hmac_sha1(r5, s0, r6, r3, r8);
            r2 = 0x13;
            b64_encode(r8, r7, r2);
            r3 = *r4;
            r0 = r4;
            r1 = r7;
            r3 = *(r3 + 0x29c);
    }
    else {
            r3 = *r4;
            r0 = r4;
            r3 = *(r3 + 0x29c);
            r1 = 0x3d4d;
    }
    r0 = (r3)(r0, r1, r2, r3);
    return r0;
}

Porem comparando com a versão anterior do aplicativo do sinesp, o código não muda, é o mesmo.

**Comment by rafaelrabaco** [comment]
@marcelocecin criei o grupo no telegram, https://telegram.me/joinchat/CX4rEwaWKYsheeW3h4xgKA

**Comment by lrenato12** [comment]
Teriam o link do grupo do telegran? Esse está quebrado ou expirou.

Preciso do web service do Sinesp Veículo, alguém tem atualizado?
